r124/outref/ydb312_gtm8182a.txt

```diff
@@ -0,0 +1,28 @@
+# Create the DB
+
+
+# start INST1-INST2 and INST3-INST4 connections
+==Executing MULTISITE_REPLIC 'START INST1 INST2'==
+Starting Primary Source Server in ##TEST_PATH##
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance2
+==Executing MULTISITE_REPLIC 'START INST3 INST4'==
+Starting Primary Source Server in ##FILTERED##_REMOTE_TEST_PATH_/instance3
+Starting Passive Source Server and Receiver Server in ##FILTERED##_REMOTE_TEST_PATH_/instance4
+
+# run updateINST3^ydb312gtm8182a.m to write variable to INST3 DB to read later
+
+# run ydb312gtm8182a.m to:
+#  	-update INST1
+#  	-read from INST3
+#  	-show attached processes of each INST JNLPOOL
```

Comment:
Add "We expect 2 processes attached to INST1 JNLPOOL and only 1 process attached to INST3 JNLPOOL".
Also add comment that the source server counts as 1 process in the above so this verifies that the mumps process only attached to INST1 JNLPOOL (for update) and not to INST3 JNLPOOL (for read).